src/SpecFlow.NetCore/AppConfig.cs

```diff
@@ -52,16 +54,34 @@ public static AppConfig CreateIn(DirectoryInfo directory)
 			return config;
 		}
 
-		public void Validate()
+		private static string GuessUnitTestProvider(DirectoryInfo directory)
```

Comment:
I'm in two minds about guessing the provider. I like the convenience but I also like being explicit; if we can take it as an argument then we don't need to depend on `Newtonsoft.Json`.